lib/tagged_content_exporter.rb

```diff
@@ -0,0 +1,72 @@
+class TaggedContentExporter
+  TRANSPORT_TAXON_ID = "a4038b29-b332-4f13-98b1-1c9709e216bc".freeze
+
+  def self.call
+    new.content_items_with_taxons.compact
+  end
+
+  def initialize
+    @content_items = ProjectContentItem
+      .for_taxonomy_branch(TRANSPORT_TAXON_ID)
+      .done
+  end
+
+  def content_items_with_taxons
+    @content_items.map do |content_item|
+      {
+        content_id: content_item.content_id,
+        url: content_item.url.gsub("https://www.gov.uk", ""),
```

Comment:
You could use Plek here, to make this environment independent.

Reply:
Amended and rebased